src/components/CustomSearch.vue

```diff
@@ -0,0 +1,39 @@
+<template>
```

Comment:
This component is useless, please finish it and use it

Reply:
why it is useless ?
we use it at home and at the help page?
reusebale components are not useless..

Reply:
Sorry for being so harsh, you just hardcoded the component. In general I call hardcoded components useless. You cannot put any data in it, neither listen to events from inside.

If you want I can take this branch over tomorrow. Please don't land it in the current state.